components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -31,7 +33,10 @@ var credentialHelper = &cobra.Command{
 	Long:   "Supports reading of credentials per host.",
 	Args:   cobra.MinimumNArgs(1),
 	Hidden: true,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Why do we need to change semantic of this command? 

If it is to track system errors to GCP then I think we should silent them to a user as before.